### PR TITLE
Feature/rework requester secrets

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

File: for-developers/confidential-computing/app-developer-secret.md [hunk 1]
```diff
@@ -277,4 +273,5 @@ At this stage, your application is ready to be tested on iExec with the followin
 
 ## Next step?
 
-Thanks to the explained confidential computing workflow, it is possible to use an app developer secret with a trusted application.
+Thanks to the explained confidential computing workflow, you discovered how to use an app developer secret with a trusted application.
+Check out how to [Access requester secrets](requester-secrets.md).
```
Comment (james-toussaint) [inline for-developers/confidential-computing/app-developer-secret.md]:
```suggestion
Check out how to:
- [Access requester secrets](requester-secrets.md)
- [Access a confidential dataset](sgx-encrypted-dataset.md)
```

File: for-developers/confidential-computing/requester-secrets.md [hunk 2]
```diff
@@ -11,12 +11,16 @@ Before going any further, make sure you managed to [Build with a TEE framework](
...
 - Familiarity with the basic concepts of [Intel® SGX](intel-sgx-technology.md#intel-r-software-guard-extension-intel-r-sgx) and [SCONE](intel-sgx-technology.md#scone-framework) framework.
-  {% endhint %}
+{% endhint %}
 
 Trusted Execution Environments offer a huge advantage from a security perspective. They guarantee that the behavior of execution does not change even when launched on an untrusted remote machine. The data inside this type of environment is also protected, which allows its monetization while preventing leakage.
 
 With iExec, it is possible to securely consume requester-provided secrets in the application.
 
+{% hint title="info" %}
+In this tutorial, you will learn how to leverage requester secrets by using the `IEXEC_REQUESTER_SECRET_#` environment variables in your code.
```
Comment (james-toussaint) [inline for-developers/confidential-computing/requester-secrets.md]:
```suggestion
In this tutorial, you will learn how to leverage requester secrets by using following environment variables in your code:
- `IEXEC_REQUESTER_SECRET_1`
- `IEXEC_REQUESTER_SECRET_2`
- `...`
- `IEXEC_REQUESTER_SECRET_<N>`
```

File: for-developers/confidential-computing/requester-secrets.md [hunk 3]
```diff
@@ -185,193 +195,111 @@ COPY ./src /app
...
+
+### Check secrets availability in the SMS
+
+```bash
 iexec requester check-secret my-namespace --chain bellecour
 iexec requester check-secret my-key --chain bellecour
 ```
 
-```sh
+```bash
```
Comment (james-toussaint) [inline for-developers/confidential-computing/requester-secrets.md]: To remove right?

File: for-developers/confidential-computing/requester-secrets.md [hunk 4]
```diff
@@ -185,193 +195,111 @@ COPY ./src /app
...
 iexec requester push-secret my-namespace --chain bellecour
 iexec requester push-secret my-key --chain bellecour
-# check the secret is available on the SMS
+```
+
+### Check secrets availability in the SMS
+
+```bash
 iexec requester check-secret my-namespace --chain bellecour
```
Comment (james-toussaint) [inline for-developers/confidential-computing/requester-secrets.md:289]: In a next PR we should change secret names to something like:
```
my-phone-number
my-credit-card-number
```

File: for-developers/confidential-computing/requester-secrets.md (hunk 2 above)
Comment (jbern0rd) [reply for-developers/confidential-computing/requester-secrets.md]: Thanks, updated in 0606d5a

File: for-developers/confidential-computing/app-developer-secret.md (hunk 1 above)
Comment (jbern0rd) [reply for-developers/confidential-computing/app-developer-secret.md]: Done in e9588cf.
I added link to result encryption tutorial as well.

File: for-developers/confidential-computing/requester-secrets.md (hunk 3 above)
Comment (jbern0rd) [reply for-developers/confidential-computing/requester-secrets.md]: Removed in 5daec3a.
I tried to improve the chain.json edition part.
Please, have a look

File: for-developers/confidential-computing/requester-secrets.md (hunk 4 above)
Comment (jbern0rd) [reply for-developers/confidential-computing/requester-secrets.md:289]: Those new names feel weird to me. They do not match the provided countapi application.
Maybe we should rename them to `count-api-namespace` and `count-api-key` ?

Comment (jbern0rd) [reply for-developers/confidential-computing/requester-secrets.md:289]: See this in the tutorial python source code
```
    const secretNamespace = process.env.IEXEC_REQUESTER_SECRET_1;
    const secretKey = process.env.IEXEC_REQUESTER_SECRET_2;
```

File: for-developers/confidential-computing/app-developer-secret.md (hunk 1 above)
Comment (james-toussaint) [reply for-developers/confidential-computing/app-developer-secret.md]: Thanks!

File: for-developers/confidential-computing/requester-secrets.md (hunk 3 above)
Comment (james-toussaint) [reply for-developers/confidential-computing/requester-secrets.md]: Thanks, please see https://github.com/iExecBlockchainComputing/documentation/pull/126/commits/5092fcb0a3d06fc00170fab186f6377ec431c437

File: for-developers/confidential-computing/requester-secrets.md (hunk 4 above)
Comment (james-toussaint) [reply for-developers/confidential-computing/requester-secrets.md:289]: It was just an example, a more preciser answer is:
```
  --secret 1=my-firstname \
  --secret 2=my-lastname\
```
`https://api.countapi.xyz/hit/${secretLastName}/${secretFistName}`
`https://api.countapi.xyz/hit/doe/john`

File: for-developers/confidential-computing/requester-secrets.md (hunk 3 above)
Comment (jbern0rd) [reply for-developers/confidential-computing/requester-secrets.md]: Thanks for this update

File: for-developers/confidential-computing/requester-secrets.md (hunk 4 above)
Comment (jbern0rd) [reply for-developers/confidential-computing/requester-secrets.md:289]: Why not, we can discuss this. Current names match the countapi documentation. They can be changed for other names as well.

**Review by james-toussaint — APPROVED** [review]
:+1:

**Pull request merged**